plugins/woocommerce/tests/php/src/Internal/FraudProtection/ApiClientTest.php

```
@@ -48,436 +52,295 @@ public function tearDown(): void {
+	 *
+	 * @testdox verify() calls Blackbox API /verify endpoint with the correct payload
 	 */
-	public function test_send_event_returns_allow_when_blog_id_not_found(): void {
-		update_option( 'jetpack_options', array( 'id' => null ) );
+	public function test_verify_calls_verify_endpoint(): void {
+		$captured_url  = null;
+		$captured_body = null;
+
+		add_filter(
+			'pre_http_request',
+			function ( $preempt, $args, $url ) use ( &$captured_url, &$captured_body ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$preempt'. (undefined)

(UnusedFormalParameter)